pyof/foundation/basic_types.py

```diff
@@ -237,18 +242,20 @@ class IPAddress(GenericType):
     netmask = UBInt32()
     max_prefix = UBInt32(32)
 
-    def __init__(self, address="0.0.0.0/32"):
+    def __init__(self, value=None):
```

Comment:
About consistency, you are right. The project started using None in all attributes. Later, we needed default values and decided to write it in the constructor declaration. It's not in our priorities to make this change in older code and that's why the code is how it is.

We started to set default immutable arguments together with constructor parameters because of better readability without the need to add extra lines of code. But I didn't understand why `Object(None)` is important. Do you have a practical example where `Object(None)` would be the only viable solution or a reasonably better one?